tasks/libs/github_actions_tools.py

```diff
@@ -5,6 +5,7 @@
 from datetime import datetime, timedelta
 from time import sleep
 
+import requests
```

Comment:
Small note here: in the past, all `requests` imports in the `tasks/` directory were done only when needed (see for instance `RemoteAPI.make_request`), because setting `import requests` at the top level would run it anytime _any_ invoke task is run, and some jobs that use invoke tasks in this repository did not have `requests` installed by default (because they didn't need it).

I don't know if that is still the case today, but if it is, then merging this PR as-is would break this.

Reply:
Oh I did not know this, I will include the `import` instruction in the `download_artifacts_with_retry` then